modules/support-generating-a-sosreport-archive.adoc

```diff
@@ -85,6 +85,13 @@ If an existing `toolbox` pod is already running, the `toolbox` command outputs `
 ----
 <1> `-k` enables you to define `sosreport` plugin parameters outside of the defaults.
 +
+.. Optional: run the following command to include all the information in from a node in your report:
```

Comment:
```suggestion
.. Optional: run the following command to include all the information which includes all rotated/compressed  libovsdb logs from a node in your report:
```

Reply:
note that the term "libovsdb" is not user facing and hence should be avoided in user facing docs.

Reply:
Does the updated wording work?